app/_src/production/dp-config/transparent-proxying.md

```diff
@@ -36,26 +36,45 @@ Prerequisites:
 {{site.mesh_product_name}} comes with [`kumactl` executable](/docs/{{ page.version }}/explore/cli) which can help us to prepare the host. Due to the wide variety of Linux setup options, these steps may vary and may need to be adjusted for the specifics of the particular deployment.
 The host that will run the `kuma-dp` process in transparent proxying mode needs to be prepared with the following steps executed as `root`:
 
-1. Create a new dedicated user on the machine.
+1. Use proper version of iptables
+    
+    {{site.mesh_product_name}} [isn't yet compatible](https://github.com/kumahq/kuma/issues/8293) with `nf_tables`. You can check the version of iptables with the following command
+    ```sh
+    $ iptables --version
+    iptables v1.8.7 (nf_tables)
+    ```
+    
+    On the recent versions of Ubuntu, you need to change default `iptables`.
+    
+    ```sh
+    $ update-alternatives --set iptables /usr/sbin/iptables-legacy
+    $ update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
+    $ iptables --version
+    iptables v1.8.7 (legacy)
+```
```

Comment:
```suggestion
    ```sh
    update-alternatives --set iptables /usr/sbin/iptables-legacy
    update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
    iptables --version
    # iptables v1.8.7 (legacy)
    ```
```
It allows to copy and paste the code block without need to remove anything before running it in shell